### PR TITLE
Add new image and styles on partner page

### PR DESCRIPTION
**Pull request by hectoruch**
Task => https://www.pivotaltracker.com/story/show/130431921

### CODE REVIEW

File: app/styles/components/c-other-section.scss [hunk 1]
```diff
@@ -18,13 +18,36 @@
...
+    flex-direction: column;
+
+    @media #{$mq-mobile-large} {
+      flex-direction: row;
       padding-left: 90px;
       padding-right: 90px;
     }
 
     img {
+      margin-bottom: 20px;
```
Comment (clementprdhomme) [inline app/styles/components/c-other-section.scss]: Remove this margin in the media query because on desktop, the logo are not vertically centred.

**Comment by hectoruch** [comment]
Fix and ready to review @clementprdhomme

**Comment by hectoruch** [comment]
Ready to review @clementprdhomme

**Comment by tiagojsag** [comment]
Logos have a huge on the left around 450px width.

**Comment by tiagojsag** [comment]
Looks ok to me. @clementprdhomme is this ok by you?

**Pull request merged**